src/tests/fhir/resolvers.test.ts

```diff
@@ -111,4 +123,345 @@ describe('fhir phoneContactResolver', () => {
         expect(phoneContactResolver(input1)).toEqual(expected1)
     })
     // XXX Add more testcases as real world data is found
+})
+
+describe('fhir hprNumberFromIdentifiers', () => {
+    test('a identifiers array with empty hpr number, but others populated', () => {
+        const identifiers: IIdentifier[] = [
+            {
+                "use": IdentifierUseKind._official,
+                "system": "urn:oid:1.3.6.1.4.1.9038.51",
```

Comment:
Fint om du kan definere konstanter for urn:oid-ene, om det ikke finnes allerede for dem,

Reply:
Dette er berre eksempel data kopiert inn frå faktiske api responser for å bruke i test. 

Eg ser ikkje heilt verdien av å definere konstanter og manuelt endre testdata til å bruke dei konstantane når disse  oid nummera  ikkje er brukt nokon stad i reell systemkode. For at testen skal vere best mulig tenkje eg at det er betre å så enkelt som mulig kopiere inn dei faktiske responsane som apiet leverer.

Einaste grunnen til at "use" er endra til ein konstant er at typescript klagde viss ikkje.